### ISSUE TITLE
Missing Documentation for getGripChannel

### ISSUE THREAD

**Issue by aryaminus** [fanout/fanout-graphql-tools#23]
While `getGripChannel` is required in, there is no specific guide to how to use in it `GraphqlWsOverWebSocketOverHttpExpressMiddleware`

**Comment by gobengo** [comment]
Hello @aryaminus . I just wrote this up: https://github.com/fanout/fanout-graphql-tools/blob/master/docs/getGripChannel.md

Does it help?

Additionally, can you elaborate on where you found `getGripChannel` to be a required option? It's intended to be optional, and have a safe default. For example, [here is where it defaults to `DefaultGripChannelNamer` in `GraphqlWsOverWebSocketOverHttpExpressMiddleware`](https://github.com/fanout/fanout-graphql-tools/blob/master/src/subscriptions-transport-ws-over-http/GraphqlWsOverWebSocketOverHttpExpressMiddleware.ts#L321).

**Comment by aryaminus** [comment]
<img width="1160" alt="Screen Shot 2019-10-10 at 17 12 04" src="https://user-images.githubusercontent.com/16350823/66565021-2379bc80-eb81-11e9-8c0c-2236ee47444a.png">

Yes, I've realized that and I'm not getting why the issue persists

**Comment by gobengo** [comment]
@aryaminus What version of fanout-graphql-tools does your project depend on?

Regardless, you can at least workaround that error by providing a value like the default one in here: https://github.com/fanout/fanout-graphql-tools/blob/master/src/subscriptions-transport-ws-over-http/GraphqlWsGripChannelNamers.ts#L34

e.g.

```
const yourGetGripChannel = ({ connectionId }) => connectionId;
```

**Comment by gobengo** [comment]
@aryaminus Looks like I never published a new version after making that optional. Oops. I'll do that.

**Comment by gobengo** [comment]
@aryaminus Just published 0.2.0, which has getGripChannel as optional:https://github.com/fanout/fanout-graphql-tools/blob/v0.2.0/src/subscriptions-transport-ws-over-http/GraphqlWsOverWebSocketOverHttpExpressMiddleware.ts#L306

**Comment by aryaminus** [comment]
Thought so.  Closing then.
